app/src/userpages/components/ProfilePage/ChangePassword/changePassword.pcss

```diff
@@ -39,13 +23,92 @@
   margin-top: 1rem;
 }
 
+.footer {
+  padding: 1.5rem;
+  padding-top: 0;
+  display: grid;
+  grid-column-gap: 1rem;
+  grid-row-gap: 1rem;
+  grid-template-columns: 1fr 96px;
+}
+
+.footerText {
+  grid-row: 2;
+  grid-column: 1;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+}
+
 .forgotLink {
-  position: absolute;
-  left: 2.4rem;
-  bottom: 1.5rem;
-  z-index: 100;
+  &,
+  &:hover,
+  &:active,
+  &:focus,
+  &:visited {
+    color: #0324FF;
```

Comment:
Thinking about it again for a second… Does it overwrite [the new default styling for links](https://github.com/streamr-dev/streamr-platform/blob/666757d31c767153032319027f3b871eaa1fb323/app/src/shared/assets/stylesheets/links.css)? In other words, is it necessary?

Reply:
Good points, it's not necessary. These are elsewhere as well, will remove if I come across 🐞